### DISCUSSION TITLE
Error: Cannot find module

### DISCUSSION THREAD

**Discussion by Horlarmmy** [vercel/next.js#40111] [Help] [answered]
Hi,
can anyone please help me on this error

```
error - Error: Cannot find module '/mnt/c/Users/.../Documents/.../.next/server/pages/index.js'
Require stack:
- /mnt/c/Users/.../Documents/.../node_modules/next/dist/server/require.js
- /mnt/c/Users/.../Documents/.../node_modules/next/dist/server/next-server.js
- /mnt/c/Users/.../Documents/.../node_modules/next/dist/server/next.js
- /mnt/c/Users/.../Documents/.../node_modules/next/dist/server/lib/start-server.js
- /mnt/c/Users/.../Documents/.../node_modules/next/dist/cli/next-dev.js
- /mnt/c/Users/.../Documents/.../node_modules/next/dist/bin/next
```

**Comment by icyJoseph** [comment, marked as answer]
Have you got more context? A repository to look at? 

What command did you run and from where? Have you installed node_modules, and run build? Is it a problem during development?

## Edit

The `pages/index.tsx` file has the following component:

```tsx
  <Image src={}
                // loader={myLoader}
                className="inset-0 w-full md:justify-center h-full object-cover filter brightness-50"
                alt="main background image"
               ></Image>
```

Which is incorrect syntax, and it is leading to parsing failure, which means you cannot even build the page, and process crashes out. The exact issue is `src={}`.

A quick solution is to comment out the component, or provide a path to the `src` argument, for example:

```tsx
  <Image
            src="/images/small-eth.webp"
            // loader={myLoader}
            className="inset-0 w-full md:justify-center h-full object-cover filter brightness-50"
            alt="main background image"
            width={100}
            height={100}
          ></Image>
```

That's all there was to it, you need valid syntax.

**Reply by Horlarmmy** [reply]
I ran npm install
Then npm run Dev is giving me issue, the code was written with typescript but it's requesting for js files

**Reply by icyJoseph** [reply]
That's not necessarily the issue, all TypeScript is compiled to JavaScript to be interpreted by the runtime engine, in this case anyway.

Got a repository to look at? From which directly are you running these commands? I take it that you are using WSL?

**Reply by Horlarmmy** [reply]
Yes, I am using WSL.
https://github.com/Karlitoyo/Gaia-Hackathon

The repo link

**Reply by Horlarmmy** [reply]
I am running from the base directory

**Reply by icyJoseph** [reply]
I see, look at my updated answer. It is a syntax problem.

**Reply by Horlarmmy** [reply]
I can't find it.

**Reply by icyJoseph** [reply]
Sorry it took me a while to update the answer, you can see it now.

**Reply by icyJoseph** [reply]
did it work out? :) if so please don't forget to mark as answered.

**Comment by Horlarmmy** [comment]
Thanks it worked
